bin/ganga

```diff
@@ -55,19 +56,13 @@ def log(level, err):
         sys.stderr.write('(consider --debug option for more information)\n')
 
 try:
-    # Process options given at command line and in configuration file(s)
-    # Perform environment setup and bootstrap
-    import Ganga.Runtime
-    Ganga.Runtime._prog = Ganga.Runtime.GangaProgram()
-    Ganga.Runtime._prog.parseOptions()
-    Ganga.Runtime._prog.configure()
-    Ganga.Runtime._prog.initEnvironment()
-    Ganga.Runtime._prog.bootstrap(Ganga.Runtime._prog.interactive)
-    Ganga.Runtime._prog.new_user_wizard()
     # Import GPI and run Ganga
-    from Ganga.GPI import *
-    Ganga.Runtime._prog.run()
+    setupGanga()
+    from Ganga.Runtime import _prog
+    from Ganga import GPI
+    _prog.run(GPI.__dict__)
 except GangaException as err:
-    log(Ganga.Runtime._prog.options.force_loglevel, err)
+    from Ganga.Runtime import _prog
```

Comment:
Why do we import this in two locations, within the try and the except rather than at the top?